File: sea-shell-service-spring-web-flux/sea-shell-rest-service/src/main/java/org/jesperancinha/shell/SeaShellApplication.java
package org.jesperancinha.shell;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.web.servlet.WebMvcAutoConfiguration;
import org.springframework.boot.autoconfigure.web.servlet.error.ErrorMvcAutoConfiguration;
import org.springframework.web.reactive.config.EnableWebFlux;

@EnableWebFlux
@SpringBootApplication(exclude = {WebMvcAutoConfiguration.class, ErrorMvcAutoConfiguration.class})
public class SeaShellApplication {

    public static void main(String[] args) {
        SpringApplication.run(SeaShellApplication.class, args);
    }

}

File: sea-shell-service-spring-web-flux/src/main/kotlin/org/jesperancinha/shell/SeaShellApplication.kt
package org.jesperancinha.shell

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.webmvc.autoconfigure.WebMvcAutoConfiguration
import org.springframework.boot.webmvc.autoconfigure.error.ErrorMvcAutoConfiguration
import org.springframework.web.reactive.config.EnableWebFlux

@EnableWebFlux
@SpringBootApplication(exclude = [WebMvcAutoConfiguration::class, ErrorMvcAutoConfiguration::class])
class SeaShellApplication {

}
fun main(args: Array<String>) {
    SpringApplication.run(SeaShellApplication::class.java, *args)
}